ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/extensions/PaywallDataExtensions.kt

```diff
@@ -137,15 +136,39 @@ internal fun DefaultPaywallPreview() {
         MaterialTheme.colorScheme,
         MockResourceProvider(),
     )
-    val template2Offering = Offering(
+    return Offering(
         identifier = "Template2",
         availablePackages = availablePackages,
         metadata = mapOf(),
         paywall = paywallData,
         serverDescription = "",
     )
+}
+
+@Preview(showBackground = true, locale = "en-rUS")
+@Preview(showBackground = true, locale = "es-rES")
+@Composable
+internal fun DefaultPaywallPreview() {
+    InternalPaywall(
+        options = PaywallOptions.Builder(dismissRequest = {}).build(),
+        viewModel = MockViewModel(offering = getDefaultPreviewOffering()),
+    )
+}
+
+@Preview(showBackground = true, locale = "en-rUS", group = "footer")
```

Comment:
do we also want to add `es-rES`?

Reply:
I thought about it, but we already have one for the full screen version, and I don't think it provides much value to have one for the footer? Lmk if you think otherwise and happy to add it though.